src/status_im/mailserver/db.cljs

```diff
@@ -0,0 +1,45 @@
+(ns status-im.mailserver.db
+  (:require [status-im.mailserver.core :as mailserver]
+            [cljs.spec.alpha :as spec]))
+
+(spec/def :mailserver/state (spec/nilable #{:disconnected :connecting :added :connected :error}))
+(spec/def :mailserver/current-id (spec/nilable keyword?))
+
+(spec/def ::not-blank-string (spec/and string? seq))
+
+(spec/def :mailserver/address (spec/and string? mailserver/valid-enode-address?))
+(spec/def :mailserver/name ::not-blank-string)
+(spec/def :mailserver/id keyword?)
+(spec/def :mailserver/user-defined boolean?)
+(spec/def :mailserver/password ::not-blank-string)
+(spec/def :mailserver/sym-key-id string?)
+(spec/def :mailserver/generating-sym-key? boolean?)
+(spec/def :mailserver/mailserver (spec/keys :req-un [:mailserver/address :mailserver/name :mailserver/id]
+                                            :opt-un [:mailserver/sym-key-id
+                                                     :mailserver/generating-sym-key?
+                                                     :mailserver/user-defined
+                                                     :mailserver/password]))
+
+(spec/def :mailserver/mailservers (spec/nilable (spec/map-of keyword? (spec/map-of :mailserver/id :mailserver/mailserver))))
+
+(spec/def :request/from pos-int?)
```

Comment:
can we be more specific here and check the order of magnitude for instance (assuming that's future dates), or it doesn't make sense?

Reply:
could you elaborate I don't understand the idea

Reply:
well, i actually figured that this is not a new code, it's just moved to another file, so nevermind. What i meant is that we can define something like `:global/timestamp` and actually validate that this is a timestamp, not just `pos-int?`, or at least have an alias for this type which would make the spec more detailed. That's it. But probably not the scope of this PR.